src/settings/index.js

```diff
@@ -36,14 +48,41 @@ export default class OaiPmhSettings extends React.Component {
     this.getPage(LOGS_ROUTE, LOGS_CONFIG_NAME, Logs, 'ui-oai-pmh.logs'),
   ];
 
+
+
   render() {
+    const { intl, location : { pathname } } = this.props;
+
+    const findLabelByRoute = (path) => {
+      switch (true) {
+        case path.includes('behavior'):
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.behavior.manager' });
+        case path.includes('general'):
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.general.manager' });
+        case path.includes('sets'):
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.sets.manager' });
+        case path.includes('technical'):
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.technical.manager' });
+        case path.includes('logs'):
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.logs.manager' });
+        default:
+          return intl.formatMessage({ id: 'ui-oai-pmh.settings.title.manager' });
+      }
+    };
```

Comment:
A little easier to maintain and less copy/paste-y as that list grows:
```
const findLabelByRoute = (path) => {
  const list = ['behavior', 'general', 'sets', 'technical', 'logs'];
  return `ui-oai-pmh.settings.${list.find(i => i === path) ?? 'title'}.manager`;
};
```

Reply:
Thanks for that, fixed